addons/glsmoke/CfgAmmo.hpp

```diff
@@ -0,0 +1,9 @@
+class CfgAmmo
+{
+	class SmokeShell;
+	class G_40mm_Smoke: SmokeShell
+	{
+		simulation = "shotSmoke";
+		deflecting = 0;
```

Comment:
I don't think that `deflecting = 0;` is a good choice. Did you try slightly higher values as well?